src/view.js

```diff
@@ -18,6 +18,7 @@ var pytorch = pytorch || require('./pytorch');
 var sklearn = sklearn || require('./sklearn');
 var tf = tf || require('./tf');
 var tflite = tflite || require('./tflite');
+var openvinoIR = openvinoIR || require('./openvino-ir');
```

Comment:
`var openvino = openvino || require('./openvino');`

Reply:
renamed

is the discussion resolved now?